### PR TITLE
Fixed typos

### PR DESCRIPTION
**Pull request by rscRoman**

### CODE REVIEW

**Comment by pfreitag** [comment]
Thanks!

File: data/en/duplicate.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
...
+	"description":"Returns a clone (or deep copy) of an object or variable, leaving no reference to the original. \rUse this function to duplicate complex structures, such as nested structures and queries. When you duplicate a CFC instance, the entire CFC contents is copied, including the values of the variables in this scope at the time you call the `Duplicate` function. Thereafter, the two CFC instances are independent, and changes to one copy, for example by calling one of its functions, have no effect on the other copy. \rNote: With this function, you cannot duplicate a COM, CORBA, or JAVA object returned from the cfobject tag or the CreateObject function. If an array element or structure field is a COM, CORBA, or JAVA object, you cannot duplicate the array or structure.",
```
Comment (KamasamaK) [inline data/en/duplicate.json]: I think "in the this scope" was the intended wording, though would have been better formatted as "in the `this` scope"

Comment (pfreitag) [reply data/en/duplicate.json]: I fixed this, no pun intended: 68dad82

Comment (rscRoman) [reply data/en/duplicate.json]: Sorry for that! It's much more readable now.

**Pull request merged**